script.js

```diff
@@ -1,49 +1,64 @@
-let msgWaiting = document.querySelector(".message-waiting");
-let searchForm = document.getElementById("search-form");
-
+const msgWaiting = document.querySelector(".message-waiting");
+const searchForm = document.getElementById("search-form");
 const booksDisplay = document.getElementById("books-display");
 
-let getGoogleBooks = "https://www.googleapis.com/books/v1/volumes?q=";
-
-let apiKey = "&key=AIzaSyCw6aeDwd2srAlaiVBFSSBvyl2O7atCTaA";
+const googleBooksURL = "https://www.googleapis.com/books/v1/volumes?q=";
```

Comment:
A rule of thumb is to name functions as verbs because they _do stuff_, and variables as nouns because they _are stuff_. `getGoogleBooks` therefore sounded like a function, not a string which represents a URL.